eng/tools/spec-gen-sdk-runner/test/fixtures/specification/service2/data-plane/widget2/tspconfig.yaml

```diff
@@ -1,6 +1,6 @@
 parameters:
   "service-dir":
-    default: "sdk/widget"
+    default: "{output-dir}/sdk/widget"
```

Comment:
We should add a case where service-dir is overriden in one of the emitters to not include output-dir.